Documentation/config/core.txt

```diff
@@ -66,7 +66,11 @@ core.fsmonitor::
 	will identify all files that may have changed since the
 	requested date/time. This information is used to speed up git by
 	avoiding unnecessary processing of files that have not changed.
-	See the "fsmonitor-watchman" section of linkgit:githooks[5].
++
+See the "fsmonitor-watchman" section of linkgit:githooks[5].
++
+Note: this option is ignored if the built-in FSMonitor is enabled via the
```

Comment:
Perhaps here, just refer to the `core.useBuiltinFSMonitor` setting.

```
Note: FSMonitor hooks (and this config setting) are ignored if the built-in FSMonitor is enabled (see `core.useBuiltinFSMonitor`).
```

Reply:
Yes, much nicer.